curriculum/challenges/english/03-front-end-libraries/sass/apply-a-style-until-a-condition-is-met-with-while.english.md

```diff
@@ -29,7 +29,7 @@ tests:
   - text: Your code should set an index variable to 1 to start.
     testString: assert(code.match(/\$.*:\s*?1;/gi), 'Your code should set an index variable to 1 to start.');
   - text: Your code should increment the counter variable.
-    testString: assert(code.match(/\$(.*):\s*?\$\1\s*?\+\s*?1;/gi), 'Your code should increment the counter variable.');
+    testString: assert(code.match(/\$(.*)\s*?:\s*\$\1\s*\+\s*1;/gi), 'Your code should increment the counter variable.');
```

Comment:
```suggestion
    testString: assert(code.match(/\$(.*)\s*?:\s*\$\1\s*\+\s*1\s*;/gi), 'Your code should increment the counter variable.');
```
This is a test case I came across while testing this locally, rest everything LGTM